src/Frontpage/Frontpage.js

```diff
@@ -1,77 +1,63 @@
 import * as React from 'react';
 import './Frontpage.css';
 import DisplayDate from '../DisplayDate/DisplayDate'
-import CanClass from '../CanvasClass/CanClass';
+import CanvasClass from '../CanvasClass/CanvasClass';
+import Connect from '../Connect/Connect';
 
 const Frontpage = (props) => {
-  const players= [
-    {name: 'Thomas'},
-    {name: 'Boushra'},
-    {name: 'Gayathri'},
-    {name: 'Tahir'},
-    {name: 'Vinh'},
-    {name: 'Ingar'},
-    {name: 'Mary'},
-    {name: 'Irlend'},
-    {name: 'Maha'},
-    {name: 'Yusuf'}
-  ]
-    
-  const showPlayer = () =>{
-    let i=0
-    let index=0
-    let canvas_list=players.map(player=><CanClass playerName={player.name} key={player.name}/>)
-    let players_show_list=[]
-    //console.log(players.length)
-    
-    if(players.length%3===0){
-      while(i<(players.length/3))
-      {console.log(players.length)
-        players_show_list.push(<tr key={i}><td>{canvas_list[index]}</td><td>{canvas_list[index+1]}</td><td>{canvas_list[index+2]}</td></tr>)
-        index=index+3
-        i=i+1
+  const Url = "http://localhost:3000/api/players";
+  let players = Connect(Url);
+  const showPlayer = () => {
+    let i = 0
+    let index = 0
+    let canvas_list = players.map(player => <CanvasClass playerName={player.name} key={player.name} />)
+    let players_show_list = []
+
```

Comment:
We can also use a "coding technique" that is called "guards" (not so sure of this term though), which consists of using `if` + `return` instead og `if` `elseif` + `return`
`if (condition1) {
  // do something
  return
}
if (condition2) {
  // do something else
  return
} `
It works well when the conditions are mutually exclusive